README.md

```diff
@@ -8,23 +8,32 @@ Slack. You can visit it here: [http://greekdigitalcommunity.com](http://greekdig
 The theme is using bootstrap and is static html. To contribute please follow the
 instructions below.
 
-## Create a local copy
-First of all, please fork the repository.  
-
+## Create a branch
 On your local machine clone the base repo:
 ```bash
 git clone git@github.com:greekdigitalcommunity/greekdigitalcommunity.github.io.git
 ```
-Set up the remotes:
+
+If you are contributing on an issue create a branch from develop like so:
 ```bash
-git remote set-url upstream git@github.com:greekdigitalcommunity/greekdigitalcommunity.github.io.git
-git remote set-url origin <your_forks_giturl>
+git checkout -b issue#XX--name_of_the_issue_you_are_implementing develop
```

Comment:
Remove “develop” word - no need I think

Reply:
It ensures that they branch from develop.